### DISCUSSION TITLE
It's necessary to download the file to view the pdf

### DISCUSSION THREAD

**Discussion by GabiGD** [requarks/wiki#5442] [Error / Bug Report]
Hi! Some days ago, it was possible to view the pdfs added in the wiki itself, but now you can view them only if you dowload the file. Does anybody know how to solve this problem? Tks.

**Comment by phxfoto** [comment]
I am curious about this too as we have many PDFs on our site for safety documentation or equipment manuals. I would like to have a way to just view the link rather than download it to every system that clicks the link.
